lang/lang-en.js

```diff
@@ -513,5 +553,5 @@
     TypeError: 'Your webcam is not initialized properly. Please reload the page.',
     newPage: 'Insert page',
   };
-  window.message = message;
+  window.congreaLanguages.en = message;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to property of function parameter 'window'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)